io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs

```diff
@@ -0,0 +1,62 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
should be 2020?

Reply:
Resolved in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/6/commits/0385a6d8936ff3b7c647d680cf0b9a29f3ebc140